src/toolkits/drawing_classifier/dc_data_iterator.cpp

```diff
@@ -0,0 +1,28 @@
+/* Copyright © 2019 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#include <toolkits/drawing_classifier/dc_data_iterator.hpp>
```

Comment:
Do we need this empty cpp file?

If we do need it, we probably don't need all these `#include` statements

Reply:
Let's not add empty `.cpp` files. Files should generally be added "lazily"